test/debug_tools_test.py

```diff
@@ -0,0 +1,48 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import os
+import subprocess
+
+import pytest
+
+import ray
+
+
+@pytest.fixture
+def ray_gdb_start():
+    # Setup environment and start ray
+    _environ = os.environ.copy()
+    for process_name in ["RAYLET", "PLASMA_STORE"]:
+        os.environ["RAY_{}_GDB".format(process_name)] = "1"
+        os.environ["RAY_{}_TMUX".format(process_name)] = "1"
+    ray.init(num_cpus=1)
```

Comment:
I'd suggest moving the `ray.init` call into the actual test inside of the setup fixture. That way, pytest will react better if the `ray.init` call fails.